readme.md

```diff
@@ -0,0 +1,3 @@
+# axisps-musicbot
+
+Just ignore this fork because it's really outdated
```

Comment:
Please revert or remove the readme.md changes as it doesn’t pertain to our repository.

Reply:
removed